test/html-to-react-tests.js

```diff
@@ -340,7 +340,7 @@ describe('Html2React', function () {
                 assert.equal(reactHtml, htmlExpected);
             });
 
-            it('should return false in case of invalid node', function () {
+            it('should return undefined in case of invalid node', function () {
```

Comment:
Not sure I understand. Do we *have to* change the public API to return undefined as opposed to false here, in order to fix this bug?